personalitytest/static/js/src/personalitytest-view.js

```diff
@@ -22,6 +22,41 @@ function PersonalityTestXBlockStudent(runtime, element) {
         });
         return buff;
     }
+    function addQuestion (question, list, count, studentAnswers) {
+        var questionInList = document.createElement('li');
+        var spanQuestion = document.createElement('div');
+        spanQuestion.className = 'question-div';
```

Comment:
why `-div` in the name – it’s already a div – as said before use more specific class names here (prefixed with the xblock’s name)